mage_integrations/mage_integrations/sources/salesforce/client/tap_salesforce/__init__.py

```diff
@@ -1,21 +1,27 @@
 #!/usr/bin/env python3
+from __future__ import annotations
```

Comment:
where is it used?

Reply:
This was added from the tap-salesforce source repo, removed in 8cc2a0e2931ffc6e60ebd0611b0fa1521525ccda